libs/api-bundle/src/Resources/config/security.php

```diff
@@ -3,9 +3,9 @@
 declare(strict_types=1);
 
 use Keboola\ApiBundle\Security\AttributeAuthenticator;
-use Keboola\ApiBundle\ServiceClient\ServiceClient;
 use Keboola\ApiBundle\Util\ControllerReflector;
 use Keboola\PermissionChecker\PermissionChecker;
+use Keboola\ServiceClient\ServiceClient;
```

Comment:
Ještě jsem si vzpomněl když jsem na to minulý týden koukal.

Chápu že tu je ten PermissionChecker a Authenticator.

Ten Service Client by ale měl být spíš někde bokem a ne v `security` ne?

Reply:
Mohl by.. a nebo by se tohle nemuselo jmenovat `security`, ale proste `services`, `api_bundle` nebo neco podobnyho. Ten bundle je prtavej, takze neni potreba (zatim?) ty definice rozdelovat.

Reply:
Ja bych to rozdelil prejmenoval, protoze to byla jedna z veci co me zmatla kdyz jsem neco hledal:)